server/database/index.js

```diff
@@ -35,6 +35,35 @@ User.hasMany(Trade, { foreignKey: 'newOwnerId' });
 // Trade.belongsTo(AssignedChores)
 // AssignedChore.hasOne(Trade)
 
+/*-----EtheriumWallet Associations-----*/
+EthereumWallet.belongsTo(User)
+User.hasOne(EthereumWallet)
+
+/*----AssignedChore Associations----*/
+AssignedChore.belongsTo(User)
+User.hasMany(AssignedChore)
+
+AssignedChore.belongsTo(Chore)
+Chore.hasMany(AssignedChore)
+
+/*-------SellChore Associations----*/
+SellChore.belongsTo(User, {as: 'originalOwner'})
+SellChore.belongsTo(User, {as: 'newOwner'})
+SellChore.belongsTo(Chore, {as: 'assignedChore'})
+
+//User.hasMany(SellChore)
+// SellChore.belongsTo(Chore)
+// Chore.hasMany(SellChore)
+
+/*-------SwapChore Associations------*/
+SwapChore.belongsTo(User, {as: 'user1'})
+SwapChore.belongsTo(User, {as: 'user2'})
+SwapChore.belongsTo(AssignedChore, {as: 'assignedChoreId1'})
```

Comment:
assignedChore1 and assignedChore2, drop the 'Id'

Reply:
dropped our the 'Id' portion of header name